# avoid comparison of pointer with integer
# error: ISO C++ forbids comparison between pointer and integer [-fpermissive]
# 
--- clients/memflush.cc.orig	2014-02-09 12:52:42.000000000 +0100
+++ clients/memflush.cc	2022-06-03 10:06:40.989938212 +0200
@@ -39,7 +39,7 @@
 {
   options_parse(argc, argv);
 
-  if (opt_servers == false)
+  if (opt_servers == NULL)
   {
     char *temp;
 
@@ -48,7 +48,7 @@
       opt_servers= strdup(temp);
     }
 
-    if (opt_servers == false)
+    if (opt_servers == NULL)
     {
       std::cerr << "No Servers provided" << std::endl;
       exit(EXIT_FAILURE);
